source/fixed-length-array.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+Create a type that represents an array of provided types and of a provided fixed length. The arrays length and Array prototype mutations that manipulate its length are excluded in the resulting type.
```

Comment:
This makes it sound like the array supports elements of various types, which is not the case.

Reply:
Check out the current description. Should be clearer.